include/beast/handler_alloc.hpp

```diff
@@ -22,12 +22,21 @@ namespace beast {
 /** An allocator that uses handler customizations.
 
     This allocator uses the handler customizations `asio_handler_allocate`
-    and `asio_handler_deallocate` to manage memory.
+    and `asio_handler_deallocate` to manage memory. It meets the requirements
+    of `Allocator` and can be used in place anywhere a `std::allocator` is
```

Comment:
Is "in place" intentional?  Would read more naturally as "...can be used anywhere a `std::`...".